src/tests/functional/plugin/shared/include/behavior/plugin/core_threading.hpp

```diff
@@ -290,6 +333,20 @@ TEST_P(CoreThreadingTestsWithIterations, smoke_LoadNetworkAccuracy) {
     }, numIterations, numThreads);
 }
 
+// tested function: single IECore ReadNetwork, SetConfig, LoadNetwork, AddExtension
+TEST_P(CoreThreadingTestsWithIterations, smoke_LoadNetwork_SingleIECore) {
+    std::atomic<unsigned int> counter{0u};
+    InferenceEngine::Core ie;
+
+    SetupNetworks();
+
+    runParallel([&] () {
+        auto value = counter++;
+        ie.SetConfig(config, deviceName);
```

Comment:
can you help explain more on what's the difference of this single IE core? from what I see, you are moving setconfig to each iteration in each threads right? do you intend to test the thread-safe of SetConfig?